spec/models/enterprise_spec.rb

```diff
@@ -386,33 +386,61 @@
       end
     end
 
-    describe "white label logo link" do
+    describe "white label logo" do
```

Comment:
I have chosen to mix the test of `white_label_logo` and `white_logo_label_link` to avoid repeating the long mock part, but I can still split it if you prefer.